### ISSUE TITLE
Following 'Getting Started'-instructions fails at 'rnpm windows'

### ISSUE THREAD

**Issue by Petter-W** [ReactWindows/react-native-windows#493]
```
PS C:\users\PettW\developer\test> rnpm windows
Installing react-native-windows@0.28.*...
npm ERR! Windows_NT 10.0.10586
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\PettW\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\n
m-cli.js" "install" "--save" "react-native-windows@0.28.*"
npm ERR! node v6.2.2
npm ERR! npm  v3.9.6

npm ERR! No compatible version found: react-native-windows@0.28.*
npm ERR! Valid install targets:
npm ERR! 1.0.0, 0.28.0-rc.1, 0.28.0-rc.0, 0.27.0, 0.27.0-rc, 0.2.5-rc7, 0.2.4, 0.2.3, 0.2.2, 0.2.1, 0.1.0
npm ERR!
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     C:\users\PettW\developer\test\npm-debug.log
child_process.js:529
    throw err;
    ^

Error: Command failed: npm install --save react-native-windows@0.28.*
npm ERR! Windows_NT 10.0.10586
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\PettW\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\n
m-cli.js" "install" "--save" "react-native-windows@0.28.*"
npm ERR! node v6.2.2
npm ERR! npm  v3.9.6

npm ERR! No compatible version found: react-native-windows@0.28.*
npm ERR! Valid install targets:
npm ERR! 1.0.0, 0.28.0-rc.1, 0.28.0-rc.0, 0.27.0, 0.27.0-rc, 0.2.5-rc7, 0.2.4, 0.2.3, 0.2.2, 0.2.1, 0.1.0
npm ERR!
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     C:\users\PettW\developer\test\npm-debug.log

    at checkExecSyncError (child_process.js:486:13)
    at Object.execSync (child_process.js:526:13)
    at Object.windows [as func] (C:\Users\PettW\Developer\test\node_modules\rnpm-plugin-windows\src\windows.js:74:16)
    at Command.runAction (C:\Users\PettW\AppData\Roaming\npm\node_modules\rnpm\bin\cli:23:15)
    at Command.listener (C:\Users\PettW\AppData\Roaming\npm\node_modules\rnpm\node_modules\commander\index.js:301:8)
    at emitTwo (events.js:106:13)
    at Command.emit (events.js:191:7)
    at Command.parseArgs (C:\Users\PettW\AppData\Roaming\npm\node_modules\rnpm\node_modules\commander\index.js:615:12)
    at Command.parse (C:\Users\PettW\AppData\Roaming\npm\node_modules\rnpm\node_modules\commander\index.js:458:21)
    at Object.<anonymous> (C:\Users\PettW\AppData\Roaming\npm\node_modules\rnpm\bin\cli:37:5)
 PS C:\users\PettW\developer\test>
```

**Comment by rozele** [comment]
Thanks @Petter-W. The problem is we haven't published version 0.28.0 of react-native-windows.  You should be able to get around this by running `rnpm windows --windowsVersion 0.28.0-rc.0`.

**Comment by rozele** [comment]
Just published 0.28.0, this should no longer be an issue.
